MARC.js

```diff
@@ -541,7 +542,26 @@ record.prototype.translate = function(item) {
 		this._associateNotes(item, "545", "ab");
 		
 		// Extract title
-		this._associateDBField(item, "245", "abn", "title");
+		// a = main title
+		// b = subtitle
+		// n = volume title
+		var titlesubfields = this.getFieldSubfields("245")[0];
+		item.title = titlesubfields["a"];
+		if (titlesubfields["b"]) {
+			//If the title did not end in a punctuation mark, we should add a colon
+			if (item.title.search(/[\?:,\.!;]\s*$/) == -1) {
+				item.title += ':';
+			}
+			item.title += ' ' + titlesubfields["b"];
+		}
+		if (titlesubfields["n"]) {
```

Comment:
And there was a reason we didn't treat it as volume, right? (Sorry again for making you repeat this, but I'd have to dig up our old discussion).